documentation/create_overview_doc/README.md

```diff
@@ -19,17 +19,17 @@ This powerful tool streamlines the creation of high-level documentation for soft
 
 - **Auto-Retry Logic and Enhanced Error Handling**: Ensures reliable documentation generation even in unstable environments through sophisticated error handling and retry mechanisms.
 
-- **Visualization with Mermaid.js Flow Maps and Enhanced Syntax Handling**: AI-generated visual flow maps to represent module interactions. Now with improved handling of Mermaid diagram syntax, minimizing manual interventions.
-
-- **Mermaid CLI Image Conversion**: Transforms Mermaid.js diagrams into images, offering a clear visual representation of your code architecture.
+- **Visualization with Code2Flow**: AI-generated visual flow maps to represent module interactions, configured with flags such as --hide-legend, --no-trimming, --verbose, and --skip-parse-errors for optimal detail and clarity.
 
+- **Code2Flow CLI Image Conversion**: The tool uses Code2Flow for creating detailed flow maps, offering a clear visual representation of your code architecture.
+- **Customizable Flowchart Generation: Code2Flow** Configured with flags such as --hide-legend, --no-trimming, --verbose, and --skip-parse-errors to optimize the detail and clarity of flowcharts.**
 - **Skippable Files and Directories**: Customize which files and directories to ignore during documentation via the `skip_list.csv`.
 
 - **Comprehensive Documentation**: Generates detailed markdown files for each module and compiles them into an overarching High_Level_Doc, complete with visual flow maps. 
 
 - **Documentation Metrics Logging**: Track your session duration and token usage metrics, recorded in `bito_usage_log.txt`.
 
-- **Required Tool and File Verification**: Checks for the presence of necessary tools ("bito", "mmdc") and prompt files before starting the documentation process.
+- **Required Tool and File Verification**: Checks for the presence of necessary tools ("bito", "code2flow", "dot") and prompt files before starting the documentation process.
```

Comment:
**Code Structure Issue**: The transition from Mermaid.js to Code2Flow in the documentation is not clearly explained, potentially leading to confusion. <br> **Fix**: Add a brief explanation of why Code2Flow was chosen over Mermaid.js and how it benefits the project to the README.md file. <br> **Code Suggestion**: 
 ```
 - **Visualization with Code2Flow**: AI-generated visual flow maps to represent module interactions, configured with flags such as --hide-legend, --no-trimming, --verbose, and --skip-parse-errors for optimal detail and clarity.

 + **Reason for Transition to Code2Flow**: The project has transitioned from using Mermaid.js to Code2Flow for flowchart generation to leverage advanced features and improved scalability. Code2Flow offers enhanced detail and clarity in flowcharts, making it easier to represent complex module interactions within our project.
 ```